.circleci/config.yml

```diff
@@ -154,6 +154,24 @@ commands:
           template: basic_fail_1
           branch_pattern: develop
           mentions: "<< parameters.mentions >>"
+      - run:
```

Comment:
as a followup improvement, it'd be nice to put that logic into a proper orb (or adopt an existing one?)

Reply:
Yes, agreed.